src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesConfiguration.kt

```diff
@@ -0,0 +1,39 @@
+package uk.gov.justice.digital.hmpps.prisonerfromnomismigration.activities
+
+import org.springframework.beans.factory.annotation.Qualifier
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.security.oauth2.client.ReactiveOAuth2AuthorizedClientManager
+import org.springframework.security.oauth2.client.web.reactive.function.client.ServerOAuth2AuthorizedClientExchangeFilterFunction
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import uk.gov.justice.digital.hmpps.prisonerfromnomismigration.health.HealthCheck
+
+@Configuration
+class ActivitiesConfiguration(
```

Comment:
Should this be Appointments?

Reply:
Good Q - appointments endpoints are actually in the activities api